### PR TITLE
release 2.4.1

### PR DESCRIPTION
**Pull request by bmc-msft**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,18 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## 2.4.1
+### Changed
+* Agent: Verifying LibFuzzer targets at the start of a task using `-help=1` now happens prior to sending heartbeats.  [#528](https://github.com/microsoft/oneufzz/pull/528)
```
Comment (ranweiler) [inline CHANGELOG.md:9]:
```suggestion
* Agent: Verifying LibFuzzer targets at the start of a task using `-help=1` now happens prior to sending heartbeats.  [#528](https://github.com/microsoft/onefuzz/pull/528)
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,18 @@ All notable changes to this project will be documented in this file.
...
+
+### Fixed
+* Service: Fixed issue related to Azure Functions not always providing the JWT token via Authorization headers. [#522](https://github.com/microsoft/onefuzz/pull/522)
```
Comment (ranweiler) [inline CHANGELOG.md:12]: Wrong PR?

**Review by ranweiler — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (bmc-msft) [reply CHANGELOG.md]: Yep.  More coffee needed.  Thanks for checking these.

File: CHANGELOG.md (hunk 1 above)
Comment (bmc-msft) [reply CHANGELOG.md]: yep, cut and paste this from a previous entry, which was also wrong.

**Review by ranweiler — APPROVED** [review]

**Pull request merged**